spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/AtlasClient.scala

```diff
@@ -33,6 +33,8 @@ trait AtlasClient extends Logging {
 
   def updateAtlasTypeDefs(typeDefs: AtlasTypesDef): Unit
 
+  def findEntity(typeNang: String, qualifiedName: String): AtlasEntity
```

Comment:
This is needed because it should associate with existing Hive tables.

Reply:
This is currently only used at https://github.com/hortonworks-spark/spark-atlas-connector/pull/125/files#diff-c9f2d7aa788ac902a55c967394916cebR269